apps/web/lego-moc-instructions-app/.env.example

```
@@ -6,13 +6,13 @@
 # =============================================================================
 
 # Main LEGO Projects API (native)
-VITE_API_BASE_URL=http://localhost:3001
+VITE_API_BASE_URL=http://localhost:${LEGO_API_PORT:-9000}
```

dotenv-linter (3.3.0)

[warning] [SubstitutionKey] The VITE_API_BASE_URL key is not assigned properly

(SubstitutionKey)